cypress/support/index.js

```diff
@@ -16,5 +16,7 @@
 // Import commands.js using ES2015 syntax:
 import './commands'
 
+require('cypress-xpath')
+
```

Comment:
Would it work with `import`?

```suggestion
import 'cypress-xpath'

```

Reply:
It works with 'import' as well. I referenced cypress-xpath plugin documentation again, and it uses 'require'. I did some research, and apparently 'import' is a ES6 version for 'require'.  I will change to import.